app/patients/edit/controller.js

```diff
@@ -517,6 +517,12 @@ export default AbstractEditController.extend(BloodTypes, ReturnTo, UserSession,
   },
 
   afterUpdate: function(record) {
+    var patient = this.get('model');
+    if (patient.getProperties('createOutpatientVisit').createOutpatientVisit) {
+      var visits = this.get('model.visits');
```

Comment:
visits is always going to be null in this situation, so instead of
```javascript
      var visits = this.get('model.visits');
```
use this:
```javascript
      var visits = [];
```

This will fix the error thrown in the tests.